srttools/monitor/conftest.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+try:
+    import tornado
+    import watchdog
+
+    MONITOR_DEPENDENCIES = True
+except ImportError:
+    MONITOR_DEPENDENCIES = False
+
+
+def pytest_ignore_collect(collection_path):
+    if MONITOR_DEPENDENCIES:
+        return False
+    return True
```

Comment:
This file tells pytest to skip the collection of the srttools/monitor directory when `tornado` and `watchdog` dependencies are not installed. I did this in order to change the behavior of the failed imports in the monitor.py and webserver.py modules. I didn't particularly like the old `= object` approach. With this, the CLI can still be executed and it will exit with 1 whenever any dependency is not installed, printing the warning to the user. Whenever the Monitor class is tried to be imported with missing dependencies, a ImportError is raised instead of just a warning.
Unfortunately pytest raises an error if in the collecting phase a ImportError is raised, stopping the procedure. This patch avoids that and monitor tests that require no dependecies can still be executed regularly.